spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -193,9 +196,9 @@ def __init__(self, resfile_path: PathType = None, clufile_path: PathType = None,
             assert folder_path.is_dir(), 'The folder_path must be a directory!'
 
             res_files = [f for f in folder_path.iterdir() if f.is_file()
-                         and '.res' in f.name and '.temp.' not in f.name]
+                         and '.res' in f.suffixes and '.temp' not in f.suffixes and len(f.suffixes) <= 2]
```

Comment:
The newly discovered format is of the type 'session_id.res.%d.MM.DD.YYYY.HH.MM', but also exists alongside the actual well-formatted 'session_id.res.%d' that we want.

The new error check covers the previous '.temp' case as well as this.